src/scripts/data_loader.py

```diff
@@ -192,8 +192,12 @@ def load_biblio(self, file_name, preload_ids=False, chunksize=1000):
         sql_alc_conn = self.db.connect()
         db_api_conn = sql_alc_conn.connection
 
-        title_ins = DBBatcher(db_api_conn, 'insert into schembl_document_title (schembl_doc_id, lang, text) values (:1, :2, :3)')
-        classes_ins = DBBatcher(db_api_conn, 'insert into schembl_document_class (schembl_doc_id, class, system) values (:1, :2, :3)')
+        if ("cx_oracle" in str(self.db.dialect)):
+            title_ins = DBBatcher(db_api_conn, 'insert into schembl_document_title (schembl_doc_id, lang, text) values (:1, :2, :3)')
+            classes_ins = DBBatcher(db_api_conn, 'insert into schembl_document_class (schembl_doc_id, class, system) values (:1, :2, :3)')
+        else:
+            title_ins = DBBatcher(db_api_conn, 'insert into schembl_document_title (schembl_doc_id, lang, text) values (%s, %s, %s)')
```

Comment:
Would it be possible to refactor this code a little, to avoid the duplication? Basically I'm a big fan of the DRY principle, and I think the duplication of the SQL is unnecessary here. 

Instead, I'd suggest using the if/else block to define the parameter substitution strings, perhaps using a helper function based on the number of parameters required. That way, the definition of "title_ins" and "classes_ins" will be cleaner.